static/js/server_events_dispatch.js

```diff
@@ -371,6 +372,7 @@ exports.dispatch_normal_event = function dispatch_normal_event(event) {
             for (const rec of event.subscriptions) {
                 const sub = stream_data.get_sub_by_id(rec.stream_id);
                 stream_events.mark_unsubscribed(sub);
+                tab_bar.maybe_rerender_title_area(sub);
```

Comment:
Hmm... it looks like even `subs.update_settings_for_unsubscribed` would be better off calling `update_subscribers_ui` which means we'd drop this line.

Reply:
Ah. no.